data/templates/default.latex

```diff
@@ -156,6 +156,20 @@ $if(CJKmainfont)$
   \fi
 $endif$
 \fi
+%% Support for special Unicode characters
+% Zero width non-joiner
+\def\zerowidthnonjoiner{%
+  \texorpdfstring{%
+    \textormath{\nobreak\discretionary{-}{}{\kern.03em}%
+      \ifvmode\else\nobreak\hskip\skip11\fi}{}%
+  }{}%
+}
+\ifPDFTeX
+  \DeclareUnicodeCharacter{200C}{{\zerowidthnonjoiner}}
```

Comment:
Is there a reason for the double braces?  It seems to me that they are not needed.